linux/Battery.c

```diff
@@ -192,6 +192,8 @@ static void Battery_getSysData(double* level, ACPresence* isOnAC) {
 
    unsigned long int totalFull = 0;
    unsigned long int totalRemain = 0;
+   bool now = false;
```

Comment:
This scope change does change the logic, as now `now` does not get reset to `false` for the second and following batteries.

Reply:
Ouch you're right, I got confused by all the scopes around. Updated however now it's limited to case when full is available while now is not (which I'm affected by).